src/PartEditor/PartEditor.ts

```diff
@@ -126,6 +132,21 @@ export class PartEditorProvider implements vscode.CustomTextEditorProvider {
     }
   }
 
+  private makeDefaultPartiton() {
+    let partition: Partition = {};
+    let backends = '';
+    for (let idx = 1; idx < this._backEndNames.length; idx++) {
+      if (backends.length) {
+        backends = backends + ',';
+      }
+      backends = backends + this._backEndNames[idx];
+    };
```

Comment:
How about this one?
```suggestion
    let backends = this._backEndNames.join(",");
    };
```

Reply:
`let idx = 1;` starts from second item.
is there something like copy array from second item?

Reply:
Oh, my bad. Then, how about this one?
```typescript
let backends = this._backEndNames.slice(1, this._backEndNames.length).join(',');
```